geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseZ2Index.scala

```diff
@@ -24,18 +24,20 @@ case object HBaseZ2Index extends HBaseLikeZ2Index with HBasePlatform {
                                                sft: SimpleFeatureType,
                                                filter: HBaseFilterStrategyType,
                                                transform: Option[(String, SimpleFeatureType)]): Seq[HFilter] = {
+    import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType.RichSimpleFeatureType
     val z2Filter = Z2Index.currentProcessingValues.map { case Z2ProcessingValues(_, bounds) =>
-      configureZ2PushDown(bounds)
+      val offset = if (sft.isTableSharing) { 2 } else { 1 } // sharing + shard
```

Comment:
originally i was wondering that too haha, but no, we set table sharing to always be false for hbase: https://github.com/locationtech/geomesa/blob/master/geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStore.scala#L48
i think it's probably more consistent to keep the table sharing logic here even though it's not used, what do you think?